### DISCUSSION TITLE
Trouble installing

### DISCUSSION THREAD

**Discussion by netlaen** [lycheeverse/lychee#920] [Q&A] [answered]
Hi,
I am installing Lychee on my Ubuntu server.
Installed required php version and all dependencies, but get an error that Bcmath is missing.
However when running apt-cache search php, the bcmath module appears in the list.
Where is the missing link? Any tips?

**Comment by mre** [comment, marked as answer]
Wrong lychee. 😉 
You'll have to ask over at https://github.com/LycheeOrg/Lychee.

**Reply by netlaen** [reply]
Oh, sorry. Will do that ;)
